tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -158,7 +158,111 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    ...       Tier1
+    [Documentation]    Veerifies the item in Resource page can be marked se favourite.
+    ...                It checks if favorite items are always listed as first regardless
+    ...                the view type or sorting
+    Click Link    Resources
+    Sleep    5s
+    Sort Resources By    name
+    ${list_of_tile_ids} =    Get List Of Ids Of Tiles
+    Verify Star Icons Are Clickable    ${list_of_tile_ids}
+
+    ${range_list} =    Get Slice From List    ${list_of_tile_ids}    ${27}    ${48}
+    Add The Items In Favourites    @{range_list}
+
+    ${list_of_tile_ids} =    Get List Of Ids Of Tiles
+    Favorite Items Should Be Listed First    ${range_list}    ${list_of_tile_ids}    ${21}
+
+    Click Button    //*[@id="list-view"]
+    Sleep    2s
+    ${list_view_tiles} =    Get The List Of Ids of Tiles In List View
+    Favorite Items Should Be Listed First    ${range_list}    ${list_view_tiles}    ${21}
+
+    Click Button    //*[@id="card-view"]
+    Sleep    2s
+    Change The Sort and Check With The List    ${range_list}    type
```

Comment:
a more self-explanatory name could be `Favorite Items Should Be Listed First When Sorted By`

Reply:
Done